package.json

```diff
@@ -11,16 +11,8 @@
     "lint": "eslint --ext '.jsx,.js' ./",
     "build-babel": "babel -D --ignore '__tests__,__mocks__' packages/$PACKAGE/src -d packages/$PACKAGE/lib",
     "build-webpack": "webpack --config packages/$PACKAGE/webpack.config.js",
-    "build:playground": "PACKAGE=react-component-playground npm run build-webpack",
-    "build:component-tree": "PACKAGE=react-component-tree npm run build-babel",
-    "build:cosmos": "PACKAGE=react-cosmos npm run build-babel",
-    "build:utils": "PACKAGE=react-cosmos-utils npm run build-babel",
-    "build:webpack": "PACKAGE=react-cosmos-webpack npm run build-babel",
-    "build:dom-polyfill": "PACKAGE=react-dom-polyfill npm run build-babel",
-    "build:querystring-router": "PACKAGE=react-querystring-router npm run build-babel",
-    "build:redux-proxy": "PACKAGE=react-cosmos-redux-proxy npm run build-babel",
-    "build:context-proxy": "PACKAGE=react-cosmos-context-proxy npm run build-babel",
-    "build": "rm -rf ./packages/*/lib && npm run build:utils && npm run build:playground && npm run build:component-tree && npm run build:cosmos && npm run build:webpack && npm run build:dom-polyfill && npm run build:querystring-router && npm run build:redux-proxy && npm run build:context-proxy",
+    "build:package": "node ./scripts/build-package.js",
+    "build": "rm -rf ./packages/*/lib && npm run build:package",
```

Comment:
Passing arguments to `npm run build` was not intended. You could do that to the old target and pass `watch` to the last package `build` target. It's interesting how it turned out though, running just `build` would definitely be easier to understand 👍